noir-projects/aztec-nr/aztec/src/prelude.nr

```diff
@@ -7,7 +7,7 @@ use crate::{
     state_vars::{
     map::Map, private_immutable::PrivateImmutable, private_mutable::PrivateMutable,
     public_immutable::PublicImmutable, public_mutable::PublicMutable, private_set::PrivateSet,
-    shared_immutable::SharedImmutable, storage::Storable
+    shared_immutable::SharedImmutable, shared_mutable::SharedMutable, storage::Storable
```

Comment:
This was missing in prelude so I sneaked it in here. Noticed that when fixing conflicts in the token blacklist contract.